src/appleseed.studio/mainwindow/pythonconsole/pythonconsolewidget.cpp

```diff
@@ -321,32 +320,32 @@ void PythonConsoleWidget::open_file(const string& filepath)
 
     while (!file.eof())
     {
-        string str;
+        std::string str;
         getline(file, str);
```

Comment:
That should be qualified as well I think.

Reply:
You're right. I will also look out for other `getline`s too.